Source/WebKit/WebProcess/WebPage/DrawingAreaCocoa.mm

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This file should go into `WebPage/cocoa`

Reply:
Fixed.